src/PhpSpec/Extensions/DefaultMatchers/Extension.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace PhpSpec\Extensions\DefaultMatchers;
+
+use PhpSpec\Matcher;
+use PhpSpec\ServiceContainer;
+use PhpSpec\ServiceContainer\IndexedServiceContainer;
+
+class Extension implements \PhpSpec\Extension
```

Comment:
Maybe you could name the extension "DefaultMatchersExtension"? Just feels weird to have an implementation named the same as the contract.

Reply:
because then it'd be called `PhpSpec\Extension\DefaultMatchers\DefaultMatchersExtension` and I'd go crazy

Reply:
🤷‍♂️

Reply:
How about renaming it to `The`?

Reply:
"Oh cool, @everzet commented on a PR. I bet it's something insightful"